hbase-annotations/pom.xml

```diff
@@ -24,7 +24,7 @@
     <groupId>org.apache.hbase</groupId>
     <artifactId>hbase</artifactId>
     <version>${revision}</version>
-    <relativePath>..</relativePath>
+    <relativePath>../pom.xml</relativePath>
```

Comment:
IIRC .. is the default value for parent relativePath? It does not need to be ../pom.xml...

Reply:
In my development environment (IDEA), it really reported a red alert.  If changed to ../pom.xml, it will not report this.
<img width="527" alt="image" src="https://github.com/apache/hbase/assets/112375429/d86cfe57-ecad-49ba-9e88-3305a4cff7bb">

I found that the maven official case specifies the actual pom.xml, may be IDEA think it's more in line with the standard...
<img width="1144" alt="image" src="https://github.com/apache/hbase/assets/112375429/aee00330-d60d-4192-b231-cefd9036b718">

Reply:
OK, I found this.

https://maven.apache.org/ref/3.9.3/maven-model/maven.html#class_parent

It defaults to ../pom.xml